packages/react/src/components/Storage/FileUploader/Previewer/PreviewerIcons.tsx

```diff
@@ -1,37 +0,0 @@
-import React from 'react';
```

Comment:
These icons are in `primitives/Icon/internal` like the other internal icons now.